provider/pkg/provider/provider.go

```diff
@@ -1368,6 +1369,23 @@ func (k *kubeProvider) Diff(
 		}
 	}
 
+	if metadata.ReplaceUnready(newInputs) {
+		switch newInputs.GetKind() {
+		case "Job":
+			jobChecker := states.NewJobChecker()
+			job, err := clients.FromUnstructured(newInputs)
+			if err == nil {
```

Comment:
Is it intentional that the error is swallowed here?  Shouldn't it be returned?

Reply:
Since this error would only occur from a malformed Job object and is unrelated to the diff, I thought it made sense to ignore. I can add a comment to that effect.